### ISSUE TITLE
example with vuejs, react or angular? 

### ISSUE THREAD

**Issue by lfmunoz** [vyachkonovalov/bulma-tagsfield#6]

**Comment by crabvk** [comment]
This little lib is framework agnostic. Some modifications required to adopt it to a specific framework.

**Comment by lfmunoz** [comment]
Yes, doesn't seem easy, couldn't figure it out within 1hr. I think it would be more useful if there was an example and if it was refactored to easily be used with a framework.
